telegram/base.py

```diff
@@ -80,3 +80,23 @@ def to_dict(self):
                     data[key] = value
 
         return data
+
+    def _get_id(self):
+        for x in ['id', 'file_id', 'message_id', 'result_id', 'update_id', 'phone_number']:
+            if hasattr(self, x):
+                return self.__getattribute__(x)
+        raise NotImplementedError
+
+    def __eq__(self, other):
+        if isinstance(other, TelegramObject):
```

Comment:
From the Photosize __eq__ it has:
` if not isinstance(other, self.__class__):`
Might use that here as ' if isinstance(other, self.__class__):